src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -449,13 +466,6 @@ private void ParseVulnerability(IResultLogWriter output)
                 {
                     rule.SetProperty(_strings.Subtype, _reader.ReadElementContentAsString());
                 }
-                else if (AtStartOfNonEmpty(_strings.InstanceSeverity))
-                {
-                    if (double.TryParse(_reader.ReadElementContentAsString(), out double rank))
-                    {
-                        result.Rank = rank;
-                    }
-                }
```

Comment:
this should be removed because:

- according to Katrina, Instance severity is a deprecated property.
- we are not mapping it correctly to rank in any way since rank's range is [0,100] but InstanceSeverity ranges from [0,5] so this is wrong.